40-create-ghcred.sh

```diff
@@ -1,8 +1,30 @@
 #/bin/sh
-# create nginx config from evironment
-# *prevent envsubst from killing nginx vars with 'tr' and '@'* 
+
+# Set default value for VUE_APP_PUBLIC_PATH if not provided
+# Get VUE_APP_PUBLIC_PATH from Docker endpoint
+
+# Copy files with the specified public path
+#cp -r /usr/app/dist /usr/share/nginx/html/$VUE_APP_PUBLIC_PATH
+
+# Remove leading and trailing whitespace and slashes
+
+VUE_APP_PUBLIC_PATH=$(echo $VUE_APP_PUBLIC_PATH | xargs | sed 's/^\///;s/\/$//')
```

Comment:
since `xargs` is only used for removing whitespace you can also move this into the sed regex:
```suggestion
VUE_APP_PUBLIC_PATH=$(echo $VUE_APP_PUBLIC_PATH | sed 's/^\s*\///;s/\/\s*$//')
```